docs/index.md

```diff
@@ -92,9 +92,9 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 ## System requirements
 
-Currently, _fastlane_ is officially supported to run on 🍎 macOS. 
```

Comment:
Why remove this?

Reply:
This specific one had the Apple which @KrauseFx didn't like.
But in the complete PR the line is still there, unchanged from the original.

Reply:
Sorry, didn't see his comment.